src/components/Layout/index.tsx

```diff
@@ -2,26 +2,31 @@ import React from "react";
 import "./../../App.less";
 import { Layout } from "antd";
 import { Link } from "react-router-dom";
+import { WalletModalProvider } from "@solana/wallet-adapter-ant-design";
 
 import { LABELS } from "../../constants";
 import { AppBar } from "../AppBar";
 
 const { Header, Content } = Layout;
 
-export const AppLayout = React.memo((props: any) => {
+export const AppLayout = React.memo(({ children }) => {
   return (
-    <div className="App wormhole-bg">
-      <Layout title={LABELS.APP_TITLE}>
-        <Header className="App-Bar">
-          <Link to="/">
-            <div className="app-title">
-              <h2>Solana DAPP</h2>
-            </div>
-          </Link>
-          <AppBar />
-        </Header>
-        <Content style={{ padding: "0 50px" }}>{props.children}</Content>
-      </Layout>
-    </div>
+    <>
```

Comment:
You don't need a React fragment if there's only one child.

Reply:
This was marked as resolved but hasn't been changed?

Reply:
Sorry my bad. Fixed